en/asset-workflow/project-import.md

```diff
@@ -59,6 +59,17 @@ The time of importing depends on the size of the project. When it's importing, p
 
 The feature of importing a Cocos Studio project is developed & tested on Cocos Studio __3.10__. If you want to import an old version project, please import it to Cocos Studio 3.10 first. Then import the Cocos Studio 3.10 project to Cocos Creator.
 
+* Solve CSD nesting makes animation impossible to parse:
+
+    When the csd is nested node, will auto create a `cc.StudioComponent.PlaceHolder` component node replace it. The `nestedPrefab` attribute of the component will save nested prefab resources, which are replaced by the current node at preview.(Editor preview is not currently supported)
```

Comment:
If the csd contains a nested prefab, a dummy node will be auto created with a `cc.StudioComponent.PlaceHolder` component attached.

Reply:
at preview -> at runtime